redis/tests/test_redis.py

```diff
@@ -1,23 +1,24 @@
 import time
+from testcontainers.core.waiting_utils import wait_for_logs
 
 from testcontainers.redis import RedisContainer
 
 
 def test_docker_run_redis():
-    config = RedisContainer()
-    with config as redis:
+    with RedisContainer('redis:7.0.11') as redis:
+        wait_for_logs(redis, 'Ready to accept connections')
```

Comment:
Same, should we move this into the container implementation?

Reply:
Maybe. It depends on how we want to support these "extra" containers, as previous versions of `nginx`, `redis` etc does not necessarily have the same logs (Haven't checked for `redis`, but `nginx` is completely silent on startup up until v1.20).

If we want to only support modern versions it would be okay, and in that case we should explicitly list the version range we support. IMO this would be okay, as I believe most users only want to run a modern instance of the image, and the ones that specifically want an older version can instead specify that as its own `DockerContainer`. 

So yeah, I think we should do it and update the docs to indicate it.

Reply:
That sounds like a good plan.

Reply:
Moved logic to be part of the `start` method 👍 Updated docs along with it